events/mbed_shared_queues.cpp

```diff
@@ -57,8 +57,8 @@ EventQueue *mbed_event_queue()
 {
 #if MBED_CONF_EVENTS_SHARED_DISPATCH_FROM_APPLICATION || !defined MBED_CONF_RTOS_PRESENT
     /* Only create the EventQueue, but no dispatching thread */
-    static unsigned char queue_buffer[MBED_CONF_EVENTS_SHARED_EVENTSIZE];
-    static EventQueue queue(sizeof queue_buffer, queue_buffer);
+    static uint64_t queue_buffer[MBED_CONF_EVENTS_SHARED_EVENTSIZE];
```

Comment:
@aashishc1988 - I think you missed dividing the MBED_CONF_EVENTS_SHARED_EVENTSIZE by sizeof(uint64_t) as MBED_CONF_EVENTS_SHARED_EVENTSIZE is in number of bytes.

Reply:
Oh yes, that was careless. Fixing it now.